src/app/components/SearchResults/ResultsRow.jsx

```diff
@@ -2,46 +2,33 @@ import React from 'react';
 import {
   isEmpty as _isEmpty,
 } from 'underscore';
-import appConfig from '../../../../appConfig';
+import EBookList from '../List/EBookList';
 
 const ResultsRow = (rows) => {
   if (_isEmpty(rows.rows)) {
     return null;
   }
-  const appEnv = process.env.APP_ENV || 'production';
-  const eReaderUrl = `${appConfig.ereader[appEnv]}?url=`;
 
   const items = rows.rows.map(item => (
     {
       ebooks: (item.items) ? item.items : [],
       pub_date: (item.pub_date) ? parseInt(item.pub_date) : null,
       pub_place: (item.pub_place) ? item.pub_place : null,
       publisher: (item.publisher) ? item.publisher : null,
-      language: (item.language) ? item.language : null,
     }
   ));
 
   return (
-    <ul className="nypl-item-list">
+    <ul className="nypl-items-list">
       {
         items.map((element, key) => (
           <li className="nypl-results-item" key={key.toString()}>
             <div className="nypl-results-description">
-              <ul className="nypl-ebooks-list">
-                {
-                  element.ebooks &&
```

Comment:
You lose this logic check to check if the `ebooks` array isn't empty. Otherwise, it will always render an empty `ul` and could also throw an error if it tries to map and empty variable.

Reply:
Reworked this to use the `EBookList` component wrapped in this test to that an empty `<ul>` isn't rendered.